BrigetteDoelp/travel-tracker-solo
# Set Up Traveler Class

#### What’s this PR do?
- ​Sets up the `Traveler` class.
- Begins testing for `Traveler` class.
- Finds trips for one traveler.
- Imports `moment js`.
#### Where should the reviewer start?
​In `index.js`, moving into `Traveler.js`, then into `Traveler-test.js`.
#### How should this be manually tested?
​By running `npm test`, and opening the site through the terminal, run `npm start` and go to `localhost:8080`. Open the dev tools and make sure the appropriate data is being logged.
#### Any background context you want to provide?
​This is incomplete, and there will be data missing. 
#### What are the relevant tickets?
​Revolves #4 
​Revolves #5 
​Revolves #2 
​Revolves #6
